spec/takeaway_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'takeaway'
+
+describe TakeAway do
+
+  subject(:takeaway) {described_class.new menu_klass, sms_klass}
+  let(:menu) {double('menu', show_menu:[:pizza, 2],add: nil,dishes:{pizza: 10})}
+  let(:menu_klass) {double('menu', new: menu)}
+  let(:sms) {double('sms')}
+  let(:sms_klass) {double('sms_klass', new: sms)}
+  context 'Customer' do
+    it 'can see the menu' do
+      expect(takeaway.menu_list).not_to be_empty
+    end
+    it 'can\'t order a dish not in the menu' do
+      expect{takeaway.add_item :pomodoro, 2}.to raise_error 'not in the menu'
+    end
+    it 'can order a dish in the menu' do
+      expect{takeaway.add_item :pizza,2}.to change{takeaway.order.length}.by 1
+    end
+    it 'can\'t delete a dish not in the ordered dishes' do
+      expect{takeaway.add_item :pomodoro, 2}.to raise_error'not in the menu'
```

Comment:
Lint/SpaceBeforeFirstArg: Put space between the method name and the first argument.